rented-properties/src/main/java/org/egov/cpt/web/controllers/ReadExcelController.java

```diff
@@ -44,18 +40,20 @@ public ReadExcelController(ReadExcelService readExcelService, FileStoreUtils fil
 	@PostMapping("/read")
 	public ResponseEntity<RentDemandResponse> readExcel(@Valid @RequestBody RequestInfoWrapper requestInfoWrapper,
 			@Valid @ModelAttribute ExcelSearchCriteria searchCriteria) {
-		log.info("Start controller method readExcel()");
+		log.info("Start controller method readExcel() Request:"+searchCriteria);
 		RentDemandResponse data = new RentDemandResponse();
 		try {
 			String filePath = fileStoreUtils.fetchFileStoreUrl(searchCriteria);
 			if (StringUtils.isNotBlank(filePath)) {
-				if("0".equalsIgnoreCase(searchCriteria.getFileFormat())) {
-					data = readExcelService.getDatafromExcel(new UrlResource(filePath).getInputStream(), 0);
-				}else if("1".equalsIgnoreCase(searchCriteria.getFileFormat())) {
-					data = readExcelNewFormatService.getDatafromExcel(new UrlResource(filePath).getInputStream(), 0);
-				}
+				data = readExcelService.getDatafromExcel(new UrlResource(filePath).getInputStream(), 0);

```

Comment:
Definitely not this. We need both the formats and if format is not specified 0 should be used.